scss/base/_helpers.scss

```diff
@@ -0,0 +1,70 @@
+@mixin fix-inline-blocks($fontsize: 16px) {
+  font-size: 0;
+
+  > * {
+    display: inline-block;
+    font-size: $fontsize;
+  }
+}
+
+.cineva {
+  @include fix-inline-blocks(16px);
+}
+
+.section {
+  padding-top: 20px;
+
+  @include breakpoint($desktop) {
+    padding-top: 40px; // va aplica de la 1200 in sus
+  }
+}
```

Comment:
please remove these `.section` styles, they were only for demonstrating purposes and will cause unexpected styles for your styleguide